test/verify/machineslib.py

```diff
@@ -1134,7 +1134,11 @@ def installWithErrorTest(dialog):
         # This is applicable only for the following test so let's keep it last,
         # in order to allow the rest of the tests to run faster with QEMU KVM
         # Run modprobe -r in retry loop because it fails sometimes with 'Module kvm is in use'
-        self.machine.execute("for i in 1 2 3 4 5; do modprobe -r kvm_intel && modprobe -r kvm && break || sleep 1; done")
+        try:
+            wait(lambda: "Module kvm is in use" not in self.machine.execute("modprobe -r kvm_intel && modprobe -r kvm_intel && modprobe -r kvm"))
```

Comment:
So this needs something in addition, like a `systemctl stop pmda` (or whichever service that is), or if it's not any, a pkill ?

Reply:
Right. Just on debian stable this is not available so I had to guard the command with a || true